fastlane/Fastfile

```diff
@@ -0,0 +1,29 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+    desc "AirCastingApp"
+    lane :distribute do
+        build_app(workspace: "AirCasting.xcworkspace", scheme: "AirCasting", export_method: "ad-hoc", silent: "true", output_directory: "./build_ipa")
+         firebase_app_distribution(
+            app: "1:51291628191:ios:b090a81539b7acbb48b8aa",
+            firebase_cli_token: ENV["FASTLANE_TOKEN"],
```

Comment:
I'd change the name of the env variable from `FASTLANE_TOKEN` to `FIREBASE_TOKEN`